docs/misp2_installation_manual_18.04.md

```diff
@@ -417,6 +434,8 @@ Mobile-ID authentication setup parameters:
 # Mobile ID and its usage settings 
 mobileID.digidocServiceURL = https://digidocservice.sk.ee/ 
 mobileID.serviceName = Testimine
+mobileID.rest.trustStore.password = CHANGEME (this parameter was not commented out by default)
```

Comment:
This "(this parameter was not commented out by default)" shouldn't probably be here. The line should be:

> mobileID.rest.trustStore.password = CHANGEME